### PR TITLE
docs: create README for shared package

### PR DESCRIPTION
**Pull request by 0xyaco**
# 🤖 Linear

Closes GRT-218

## Description
* Create a README for `shared` package
* Rename:
  * `Address` service to `HexUtils`
  * `NormalizedAddress` type to `NormalizedHex`

### CODE REVIEW

File: packages/shared/src/external.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type * from "./types/index.js";
+export { HexUtils as AddressUtils, Caip2Utils, Logger } from "./services/index.js";
+export { InvalidHex as InvalidAddress, InvalidChainId } from "./exceptions/index.js";
```
Comment (0xnigir1) [inline packages/shared/src/external.ts:3]: are these `X as Y` correct? i see that in the other packages you're using HexUtils, not AddressUtils

File: packages/shared/README.md
Comment (0xnigir1) [inline packages/shared/README.md:1]: 💎

**Review by 0xnigir1 — COMMENTED** [review]
LGTM, just one comment

File: packages/shared/src/external.ts (hunk 1 above)
Comment (0xyaco) [reply packages/shared/src/external.ts:3]: Whoops, refactoring made _wrong_

File: packages/shared/README.md
Comment (0xyaco) [reply packages/shared/README.md:1]: I'm just _standing on the shoulders of READMEs/repo boilerplate giants_

**Review by 0xnigir1 — APPROVED** [review]

**Review by 0xkenj1 — APPROVED** [review]

File: packages/shared/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# ebo-agent: shared package
+
+Shared lib that for EBO agent libraries.
```
[the remark this answers is not in the document]
Comment (0xyaco) [reply packages/shared/README.md:3]: Oooh nice catch! :telescope: I'll fix it in the next docs PR

**Pull request merged**